torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -98,9 +114,16 @@ def aot_dispatch_base(
                 if maybe_subclass_meta is None
                 else maybe_subclass_meta.fw_metadata
             )
-        compiled_fw = compiler(fw_module, updated_flat_args)
 
-    # This boxed_call handling happens inside create_runtime_wrapper as well.
+        with TracingContext.report_output_strides() as fwd_output_strides:
+            compiled_fw = compiler(fw_module, updated_flat_args)
+
+        # see note: [Returning Fake Tensors on First AOT Autograd Call]
+        if tracing_context and tracing_context.fakify_first_call:
+            fakified_out = _compute_output_meta_with_inductor_strides(
+                fw_module, fwd_output_strides
```

Comment:
hmm so the invariant here is that:

(1) `tracing_context.fakify_first_call` is only true (today) while ddp_optimizer is in the middle of tracing

(2) fwd_output_strides is only set (today) if inductor was the backend.

So this code only does stuff when the two are used together

Reply:
fakified_out is still set even if inductor is not the backend, it will just be the default meta strides.